src/main/java/com/napier/sem/App.java

```diff
@@ -48,20 +52,44 @@ public void createReportFor(Region name, numberOfCountries topN) {
          * Create report.
          */
         public abstract void createReportFor();
+
```

Comment:
sorry for being picky but avoid this pointless new lines, maybe it's your editor, it just adds noise to the review. here it's fine but in a big project if your editor does this all the time, people will just ask for changes in your PR right away to fix the newline mess without even taking a look at the logic of your changes :)